packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -50,6 +50,7 @@ import {
 } from './actionHandlers';
 import VirtualScroll from '../utils/components/VirtualScroll';
 import { SELECTION_CELL_ID } from './columns';
+import { useScrollToRow } from './hooks/useScrollToRow';
```

Comment:
Can you move this import up to wear the other hooks are imported? This also means editing index.ts in `src/core/Table/hooks`.